src/components/Incident/DisplayIncident.js

```diff
@@ -1,18 +1,11 @@
-import React from 'react'
-import NotificationsIcon from 'material-ui/svg-icons/social/notifications'
-import ModeEditIcon from 'material-ui/svg-icons/editor/mode-edit'
-import Checkpoint from './Checkpoint'
-import IconButtonStyled from '../elements/IconButtonStyled'
-import BadgeStyled from '../elements/BadgeStyled'
-import Timeline from '../Timeline/Timeline'
-import Engagements from '../Engagements'
 import { CollapsibleGridSet } from '../elements/CollapsibleGrid'
-import SyncIcon from 'material-ui/svg-icons/notification/sync'
 import { connect } from 'react-redux'
+import { IncidentSummary, IncidentSummaryName } from './IncidentSummary'
+import { IncidentEvents, IncidentEventsName } from './IncidentEvents'
 
 export const DisplayIncident = ({incident, ticket, ticketSystem, expandSection, dispatch}) => {
     return CollapsibleGridSet('incident-container', 'incident-row', 'incident-col', [
-        IncidentSummary(incident, ticket, ticketSystem, null, dispatch),
+        IncidentSummary(incident, ticket, ticketSystem, ticket.originId, dispatch),
```

Comment:
Since we are already passing ticket, can we redefine the `IncidentSummary` to use that ticket for this arg?

If you do refactor it, I'd like to have this changed from ordered args to named args.

Reply:
We probably want to move the other direction: passing down ticketId and getting both ticket and originId out of state closer to where they're being used. True correction of IncidentSummary is a bit out of scope of this pull request; I just broke it into different files to make it easier to fix up later.

Refactoring to named args for now.